src/Core/src/Converters/FlexEnumsConverters.cs

```diff
@@ -218,9 +218,9 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
 			var strValue = value?.ToString();
 			if (strValue != null)
 			{
+				value = strValue.Trim();
```

Comment:
`value` does not seem to be used later. Maybe trim the `strValue`?

```suggestion
```

Reply:
👍